### DISCUSSION TITLE
Remove a sector 0, but winning PoSt will get this also

### DISCUSSION THREAD

**Discussion by yycceth** [filecoin-project/lotus#7765] [Storage Provider] [answered]
Remove a sector 0, but winning PoSt will get this, so will occur the error:
![image](https://user-images.githubusercontent.com/23360623/87657206-17183500-c78d-11ea-8857-2639e0b51024.png)

**Comment by Reiers** [comment, marked as answer]
Terminate the sector OnChain with:
`lotus-miner sectors terminate --really-do-it <sectornumber>
`

If the issue still is present - please provide more information. 
_Moving the issue to discussions._
